tools/icc_detect/icc_detect_x11.cc

```diff
@@ -3,16 +3,21 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+// clang-format off
```

Comment:
I am not sure I want to know why this is needed...

Reply:
Since the base name of the `.cc` is not quite the same as that of the `.h` (because of the `_x11`, `_win32` suffixes), clang-format thinks this is a “normal” header and wants to move it down.

Reply:
At least that is the explanation for the first one. The second one is because of that macro business in `X11/Xlib.h`…